src/service/xhr-impl.js

```diff
@@ -189,8 +203,39 @@ export class Xhr {
       init.headers['Content-Type'] = 'application/json;charset=utf-8';
       init.body = JSON.stringify(init.body);
     }
-    return this.fetch(input, init)
-        .then(response => response.json());
+
+    const isCacheable = (init.method === 'GET' && propertyPath);
+    if (isCacheable) {
```

Comment:
I bet we'll add an option to opt out of cache at some point, but this is fine :)